lib/chelsea/iq_client.rb

```diff
@@ -28,7 +28,8 @@ class IQClient
       server_url: 'http://localhost:8070',
       username: 'admin',
       auth_token: 'admin123',
-      internal_application_id: ''
+      internal_application_id: '',
+      stage: 'build'
```

Comment:
Might be good to default to `develop`, as that's more developer centric?

Reply:
I went with `build` as the default, since that is IQ's default, and is the same behavior as prior Chelsea releases.